R/article.R

```diff
@@ -490,7 +490,7 @@ sim_article <- function(..., highlight = NULL, citation_package = "natbib") {
 #' @export
 #' @rdname article
 springer_article <- function(..., keep_tex = TRUE,
-                             citation_package = "default") {
+                             citation_package = "natbib") {
   pdf_document_format(
```

Comment:
We need to add a Pandoc requirement in there as we are using some recent feature like `$it.` in template. 

You can find such R line of code in some other template

Reply:
Do you know which version is required?

Reply:
Well, GitHub Actions failed for low pandoc versions. I added 

```r
  if (!rmarkdown::pandoc_available("2.11.4")) {
    stop("`springer_article()` now requires a minimum of pandoc 2.11.4")
  }
```

Because 2.11.4 is the oldest version that didn't fail.